src/quick-view.scss

```diff
@@ -0,0 +1,100 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-quick-view;
+
+$fd-quick-view-avatar-indent: 0.75rem;
+$fd-quick-view-title-bottom-indent: 0.5rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  min-width: 20rem;
+  max-width: 30rem;
+
+  &__content {
+    @include fd-reset();
+
+    padding: 1rem;
+  }
+
+  &__header {
+    @include fd-reset();
+
+    > *:first-child {
+      border-top-right-radius: $fd-popover-border-radius;
+      border-top-left-radius: $fd-popover-border-radius;
+    }
+  }
+
+  &__subheader {
+    @include fd-reset();
+    @include fd-flex();
+
+    @include fd-rtl() {
+      > *:first-child {
+        margin-right: 0;
+        margin-left: $fd-quick-view-avatar-indent;
+      }
+    }
+
+    > *:first-child {
+      margin-right: $fd-quick-view-avatar-indent;
+    }
+
+    &-text {
+      max-width: 100%;
+      min-width: 0; // fix for flex width https://github.com/philipwalton/flexbugs#flexbug-17
+    }
+  }
+
+  &__title {
+    @include fd-reset();
+
+    font-size: var(--sapFontHeader5Size);
+    color: var(--sapTile_TitleTextColor);
+    margin-bottom: $fd-quick-view-title-bottom-indent;
+  }
```

Comment:
Let's reuse title component, instead of having it

Reply:
changed